ecosystem/cli/website.py

```diff
@@ -49,21 +57,28 @@ def _load_from_file(
     # Website strings
     web_data = toml.loads((resources_dir / "website.toml").read_text())
 
+    # Custom css
+    custom_css = None
+    if web_data.get("custom-css", False):
+        custom_css = (resources_dir / web_data["custom-css"]).read_text()
+
     # Jinja templates
     environment = Environment(loader=PackageLoader("ecosystem", "html_templates/"))
     templates = {
         "website": environment.get_template("webpage.html.jinja"),
         "card": environment.get_template("card.html.jinja"),
         "tag": environment.get_template("tag.html.jinja"),
         "link": environment.get_template("link.html.jinja"),
+        "css": environment.get_template("style.css.jinja"),
     }
-    return projects, web_data, label_descriptions, templates
+    return projects, web_data, label_descriptions, templates, custom_css
 
 
 def _build_html(projects, web_data, label_descriptions, templates) -> str:
     """
     Take all data needed to build the website and produce a HTML string.
     """
+    # pylint: disable=too-many-locals
```

Comment:
Pylint is right to complain here but I think it should be fixed by refactoring other parts of the function which is out of scope of this PR